src/ansys/mapdl/core/launcher.py

```diff
@@ -802,6 +802,11 @@ def get_start_instance(start_instance: bool = True):
     """
```

Comment:
```suggestion
    Notes
    -----
    If the environment variable ``PYMAPDL_START_INSTANCE`` is set,
    hence the argument ``start_instance`` is overwritten.
      
    """
```

Adding a note at the end of the docstring could help the user.

Reply:
Good idea!